docs/exploration-and-organization/events-and-timelines.md

```diff
@@ -12,19 +12,21 @@ Events and timelines are a way to capture that chronological knowledge and make
 
 ## Events
 
+![An event on a time series](./images/event-on-a-time-series.png)
+
 An event is basically a date + a title + a description + an icon. You can add events to Metabase to show important milestones, launches, or anything else, right alongside your data.
 
 ## Timelines
```

Comment:
Maybe mention that if you just create a bunch of events without explicitly creating a timeline, you are actually creating a timeline, as far as Metabase is concerned? This was confusing to me, because it seemed like the rest of the article applied only if I choose to proactively create a timeline, but in fact it just applies always